pages/api/v1/search/index.public.js

```diff
@@ -0,0 +1,41 @@
+import nextConnect from 'next-connect';
+
+import cacheControl from 'models/cache-control';
+import controller from 'models/controller';
+import search from 'models/search';
+import validator from 'models/validator';
+
+export default nextConnect({
+  attachParams: true,
+  onNoMatch: controller.onNoMatchHandler,
+  onError: controller.onErrorHandler,
+})
+  .use(controller.injectRequestMetadata)
+  .use(controller.logRequest)
+  .get(cacheControl.swrMaxAge(10), getValidationHandler, getHandler);
+
+function getValidationHandler(request, response, next) {
+  const cleanValues = validator(request.query, {
+    q: 'required',
+    page: 'optional',
+    per_page: 'optional',
+    sort: 'optional',
+  });
+
+  request.query = cleanValues;
+
+  next();
+}
+
+async function getHandler(request, response) {
+  const results = await search.findAll({
```

Comment:
Abaixo a variável é atribuída à outra apenas para ter o nome diferente, então pode fazer isso direto aqui:

```suggestion
  const contentListFound = await search.findAll({
```

Reply:
resolvido!